backpack/extensions/secondorder/hbp/custom_module.py

```diff
@@ -0,0 +1,10 @@
+from backpack.core.derivatives.sum_module import SumModuleDerivatives
+from backpack.extensions.secondorder.hbp.hbpbase import HBPBaseModule
+
+
+class KFACSumModule(HBPBaseModule):
+    """KFAC extension for SumModule."""
```

Comment:
Rename into `HBPSumModule`

Reply:
Fixed this in commit "Fixed doc strings, renamed KFACSumModule to HBPSumModule"